src/test/resources/application-integration-test.properties

```diff
@@ -0,0 +1,6 @@
+it.wsmUrls.dev=https://workspace.dsde-dev.broadinstitute.org/
+
+it.wsmEndpoints.createWorkspace=api/workspaces/v1
+
+it.serviceAccountEmail=william.thunderlord@test.firecloud.org
+it.serviceAccountFilePath=src/test/resources/rendered/wsm-firecloud-account.json
```

Comment:
how about moving to YAML? https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-external-config-yaml

Reply:
I discussed this at some point with (I think?) @zloery; for the time being, I don't want to have a mix of languages, but I'd be quite interested to discuss this more.

Reply:
we could adopt yaml everywhere and avoid a mix of languages ...